### PR TITLE
chore(stackdriver): Changed distribution counter naming pattern

### PR DESCRIPTION
**Pull request by ewiseblatt**
From __count to _count to be compliant to stackdriver naming policies.
@jtk54

### CODE REVIEW

File: spinnaker-monitoring-daemon/spinnaker-monitoring/stackdriver_descriptors.py [hunk 1]
```diff
@@ -320,6 +320,13 @@ def name_to_type(self, name):
     """Determine Custom Descriptor type name for the given metric type name."""
     return self.__metric_name_prefix + name
 
+  def distribution_to_counter(self, name):
+    """Converts a distribution name (or type) into a corresponding counter.
+    
+    The counter is used for convienence in determine the count in the distribution.
+    """
+    return name + '_count'
```
Comment (jtk54) [inline spinnaker-monitoring-daemon/spinnaker-monitoring/stackdriver_descriptors.py]: Previously `count` was prefixed with two underscores: `__count`. Is this change intentional?

Comment (ewiseblatt) [reply spinnaker-monitoring-daemon/spinnaker-monitoring/stackdriver_descriptors.py]: yea, that is the "stackdriver compliant naming policies" I mentioned. It is lexically valid, but the stackdriver folks frowned at the convention.

I only do this for the distribution counts, which is implicitly using the standard stackdriver policies configuration (which requires distributions instead of the individual dimension counters, which are still decorated with double-_ to ensure name disambiguation)

Comment (jtk54) [reply spinnaker-monitoring-daemon/spinnaker-monitoring/stackdriver_descriptors.py]: :+1:

**Comment by ewiseblatt** [comment]
@spinnakerbot cherry-pick 1.13
